### DISCUSSION TITLE
[react-modal-video] Requesting TS package

### DISCUSSION THREAD

**Discussion by hasanm95** [DefinitelyTyped/DefinitelyTyped#60254] [Request a new @types package]
This is a widely used react package. But, it does not have TS support.

**Comment by jbovone** [comment]
I will pick this stuff, should be quick.

**Comment by jbovone** [comment]
Update:
We have a draft for this. I leave the link to the PR. 

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60864

**Comment by hasanm95** [comment]
Hi,
How could I review this PR?

**Comment by jbovone** [comment]
Yes, you can!
